internal/cli/internal/command/volumes/volumes.go

```diff
@@ -0,0 +1,45 @@
+package volumes
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/api"
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+	"github.com/superfly/flyctl/internal/cli/internal/command/volumes/snapshots"
+)
+
+func New() *cobra.Command {
+	const (
+		long = "Commands for managing Fly Volumes associated with an application"
+
+		short = "Volume management commands"
+	)
+
+	cmd := command.New("volumes [type] <name> [flags]", short, long, nil)
+
+	cmd.Aliases = []string{"vol"}
+
+	cmd.AddCommand(
+		newCreate(),
+		newList(),
+		newDelete(),
+		newShow(),
+		snapshots.New(),
+	)
+
+	return cmd
+}
+
+func printVolume(w io.Writer, vol *api.Volume) {
```

Comment:
these are 8 `Fprintf` statements in a row. Please write everything into a buffer and then flush it once to w.